OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -187,7 +196,7 @@ public Aircraft(ActorInitializer init, AircraftInfo info)
 
 			// TODO: HACK: This is a hack until we can properly distinguish between airplane and helicopter!
 			// Or until the activities get unified enough so that it doesn't matter.
-			IsPlane = !info.CanHover;
+			IsPlane = !info.VTOL;
```

Comment:
Ok, fair enough.  I was thinking more that most of our plane activities could depend on `CanHover` and the landing on `VTOL` possibly without any further changes to the activities.  I may be wrong, however.